packages/core/admin/admin/src/StrapiApp.js

```diff
@@ -208,7 +211,10 @@ class StrapiApp {
               runHookSeries={this.runHookSeries}
             >
               <LibraryProvider components={components} fields={fields}>
-                <LanguageProvider messages={this.translations}>
+                <LanguageProvider
+                  messages={this.translations}
+                  localesNativeNames={localesNativeNames}
```

Comment:
I'm curious why "native" in the name? Why not `localeNames` ?

Reply:
The variable previously was languageNativeNames and I did not put much thought about name. I'll change it to localeNames!